kubecost/values.yaml

```diff
@@ -949,14 +949,13 @@ aggregator:
   aggregatorDbStorage:
     storageClass: ""  # default storage class
     storageRequest: 128Gi
-
   resources:
-    limits:
-      memory: 5Gi
-    requests:
-      cpu: 2000m
-      memory: 5Gi
```

Comment:
I'm in agreement with reverting the `limits`. I think we should leave the `requests` as is? What do you think?